src/bitmessageqt/foldertree.py

```diff
@@ -502,26 +485,22 @@ def data(self, role):
 
 class AddressBookCompleter(QtGui.QCompleter):
     def __init__(self):
-        super(QtGui.QCompleter, self).__init__()
+        super(AddressBookCompleter, self).__init__()
         self.cursorPos = -1
-    
+
     def onCursorPositionChanged(self, oldPos, newPos):
         if oldPos != self.cursorPos:
             self.cursorPos = -1
-        
+
     def splitPath(self, path):
-        stringList = []
-        text = unicode(path.toUtf8(), encoding="UTF-8")
-        splitIndex = rfind(text[0:self.widget().cursorPosition()], ";") + 1
-        str = text[splitIndex:self.widget().cursorPosition()]
-        str = rstrip(lstrip(str))
-        stringList.append(str)
-        return stringList
-        
+        text = unicode(path.toUtf8(), 'utf-8')
+        return [text[:self.widget().cursorPosition()].split(';')[-1].strip()]
```

Comment:
Did't you forget `lstrip`? I think that applies to situations when there is a space in front of the label.

Reply:
There is `''.strip()` instead which is supposed to replace both `rstrip` and `lstrip`